core/src/main/java/com/linecorp/armeria/common/DefaultRequestHeadersBuilder.java

```diff
@@ -153,4 +153,11 @@ public Locale selectLocale(Iterable<Locale> supportedLocales) {
         final HttpHeadersBase getters = getters();
         return getters != null ? getters.selectLocale(supportedLocales) : null;
     }
+
+    @Override
+    public Cookies cookies() {
+        final HttpHeadersBase getters = getters();
+        checkState(getters != null, ":cookie header does not exist.");
```

Comment:
- How about returning an empty `Cookie` if `getters` is null?
- JFYI, cookie is not a pseudo header. So don't need to prepend `:`. 😀

Reply:
I think it's more natural way than return `null`. I will fix them on next commit. And I didn't know that means pseudo header. Thank you for notice. '-')b